install/gcp/installer/forseti_client_installer.py

```diff
@@ -94,3 +96,38 @@ def get_deployment_values(self):
             'VPC_HOST_NETWORK': self.config.vpc_host_network,
             'VPC_HOST_SUBNETWORK': self.config.vpc_host_subnetwork
         }
+
+    def create_firewall_rules(self):
+        # Create firewall rule to open only port tcp:22 (ssh)
+        # to all the external traffic from the internet to ssh into client VM.
+        gcloud.create_firewall_rule(
+            self.format_firewall_rule_name(
+                'forseti-client-allow-ssh-external'),
+            [self.gcp_service_acct_email],
+            constants.FirewallRuleAction.ALLOW,
+            ['tcp:22'],
+            constants.FirewallRuleDirection.INGRESS,
+            0,
+            self.config.vpc_host_network,
+            '0.0.0.0/0')
+
+        # Rule to block out all the ingress traffic on client VM.
+        gcloud.create_firewall_rule(
+            self.format_firewall_rule_name('forseti-client-deny-all'),
+            [self.gcp_service_acct_email],
+            constants.FirewallRuleAction.DENY,
+            ['icmp', 'udp', 'tcp'],
+            constants.FirewallRuleDirection.INGRESS,
+            1,
+            self.config.vpc_host_network)
+
+    def format_firewall_rule_name(self, rule_name):
+        """Format firewall rule name.
+
+        Args:
+            rule_name (str): Name of the firewall rule.
+
+        Returns:
+            str: Firewall rule name.
+        """
+        return '{}-{}'.format(rule_name, self.config.identifier)
```

Comment:
This is repeated from the method in the server installer, and should be refactored.

Reply:
Server installer doesn't have create or delete functionality now. Can be refactored in the future though.

Reply:
Okay